Apps/WebClient/src/Server/Models/DependentModel.cs

```diff
@@ -0,0 +1,89 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Models
+{
+    using System.Collections.Generic;
+    using System.Text.Json.Serialization;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Database.Models;
+
+    /// <summary>
+    /// Represents a Dependent Model.
+    /// </summary>
+    public class DependentModel
+    {
+        /// <summary>
+        /// Gets or sets the dependent hdid.
+        /// </summary>
+        [JsonPropertyName("ownerId")]
+        public string OwnerId { get; set; } = string.Empty;
```

Comment:
I dont think it makes sense to have these fields

I think the name and hdid are the only fields necessary



```
public class DependentModel
{
   public string Name {}
   public string Hdid {}
public static DependentModel CreateFromPatientModel(patient model ) {
...
}
}
```

Reply:
updated